search/src/sih_logo.svg

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is not currently in use, is it?

Reply:
No it isn't; I had originally brought in an svg logo I had used for another project to test adding in a logo for the page, but decided to put that on hold for now.